apps/web/src/components/Sidebar/buildDocumentTrees.ts

```diff
@@ -0,0 +1,39 @@
+import { DocumentVersion } from '$core/schema'
```

Comment:
Use the package name. If we decide to go with Rollup refactor will be less painful. 

`@latitude-data/core`

Reply:
kind of a pita given the lsp imports automatically from $core, how can we make this seamless?